### PR TITLE
Data processing placholders description

### PR DESCRIPTION
**Pull request by vlobzakov**
https://jelastic.team/browse/JE-46528

### CODE REVIEW

File: docs/creating-manifest/placeholders.md [hunk 1]
```diff
@@ -419,6 +421,574 @@ globals:
...
+[11:40:07 CS:Placeholders.join:17]: [SUCCESS] ASSERT: [1,
+2,
+3].join(',') === '1,2,3'
+[11:40:07 CS:Placeholders.join]: END HANDLE EVENT: application/install
+[11:40:07 CS:Placeholders.join]: END INSTALLATION: CS:Placeholders - built-in data processing functions - join
+```
+
+### ${\*.toJSON()}
+
+This placeholder returns an array of words or objects in JSON format.
```
Comment (SlavaKatiukha) [inline docs/creating-manifest/placeholders.md]:
```suggestion
This placeholder returns structured data in JSON format. 
```

File: docs/creating-manifest/placeholders.md [hunk 2]
```diff
@@ -419,6 +421,574 @@ globals:
...
+  }
+}
+[12:49:40 CS:Placeholders.toJSON/print']: END HANDLE EVENT: application/install
+[12:49:40 CS:Placeholders.toJSON/print']: END INSTALLATION: CS:Placeholders - built-in data processing functions- toJSON/print'
+```
+
+
+### ${\*.contains()}
+
+This placeholder acts a regex that finds substring in a string.
```
Comment (SlavaKatiukha) [inline docs/creating-manifest/placeholders.md]:
```suggestion
This placeholder is used for checking if the specified element exists in the given list or not.  
```

File: docs/creating-manifest/placeholders.md [hunk 3]
```diff
@@ -419,6 +421,574 @@ globals:
...
+    },
+    {
+      "assert": "${globals.unknown.contains(abc)} === false"
+    }
+  ]
+}
+```
+@@!
+
+The result of such regex action is a boolean value: *true* or *false*:
```
Comment (SlavaKatiukha) [inline docs/creating-manifest/placeholders.md]:
```suggestion
The result of such action is a Boolean value: *true* or *false*:
```

File: docs/creating-manifest/placeholders.md [hunk 4]
```diff
@@ -419,6 +421,574 @@ globals:
...
+    },
+    {
+      "assert": "'${this.test2}' == 'test2'"
+    }
+  ]
+}
+```
+@@!
+
+Compare the output with this:
```
Comment (SlavaKatiukha) [inline docs/creating-manifest/placeholders.md]: confusing

**Review by SlavaKatiukha — CHANGES_REQUESTED** [review]

File: docs/creating-manifest/placeholders.md (hunk 4 above)
Comment (vlobzakov) [reply docs/creating-manifest/placeholders.md]: updated

**Review by SlavaKatiukha — APPROVED** [review]

**Pull request merged**